doc/whats_new.rst

```diff
@@ -45,6 +45,8 @@ Changelog
 Bug
 ~~~
 
+- Fix ``area_mode='range'`` in :func:`mne.viz.utils._plot_psd` Also line width set back to 0.5, und line_alpha is passed through. by `Jeff Hanna`
```

Comment:
Only reference public functions in release notes, so maybe
```
:meth:`mne.Epochs.plot_psd`
```
? Also und->and.

Finally, if all this PR does is restore previously working behavior within a given release cycle, you actually don't need to add to `whats_new.rst`. So this should only be here for example if it fixes a bug that existed with `Raw.plot_psd`.

Reply:
Yes, sorry, of course. This should now fix the titles.